src/components/admin/form-components/ListSelector.jsx

```diff
@@ -41,9 +37,9 @@ export default class ListSelector extends React.Component {
 }
 
 ListSelector.propTypes = {
-  types: PropTypes.object.isRequired,
+  elements: PropTypes.object.isRequired,
   type: PropTypes.string,
```

Comment:
Maybe we could just put a comment at the end of this type line specifying that all this type does is UI (the floating label text), type makes me think that it alters behaviour